src/main/java/docbuddy/users/model/User.java

```diff
@@ -33,4 +33,17 @@
         public static final String USERNAME = "username";
         public static final String PASSWORD = "password";
     }
+
+    @Override
+    public boolean equals(Object otherUser) {
+        if (otherUser == null) return false;
+        if (!(otherUser instanceof User))return false;
+
+        User other = (User) otherUser;
+
+        if (other == this) return true;
```

Comment:
I think I've learned from PMD, but should we have multiple returns on one method?

Link: http://www.javapractices.com/topic/TopicAction.do?Id=114

Reply:
Done.